src/main/java/roomescape/reservationTime/dao/H2ReservationTimeDao.java

```diff
@@ -0,0 +1,66 @@
+package roomescape.reservationTime.dao;
+
+import java.sql.PreparedStatement;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.boot.autoconfigure.service.connection.ConnectionDetails;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Component;
+import roomescape.common.Dao;
+import roomescape.reservationTime.ReservationTime;
+
+@Component
+public class H2ReservationTimeDao implements Dao<ReservationTime> {
+    private final JdbcTemplate jdbcTemplate;
+
+    public H2ReservationTimeDao(JdbcTemplate jdbcTemplate, ConnectionDetails connectionDetails) {
```

Comment:
```suggestion
    public H2ReservationTimeDao(JdbcTemplate jdbcTemplate) {
```
ConnectionDetails는 어떤걸 테스트하다가 추가된걸까요?

Reply:
IDE 단축키를 사용하다 잘못 추가된 걸 발견하지 못하고 놔두었던 것 같습니다..!
꼼꼼하게 확인했어야 하는 부분인데 그러지 못했네요 🤣